docs/project_ram/RAM_1pager.md

```diff
@@ -0,0 +1,41 @@
+
+As a team, we develop and deliver a range of stakeholder-related activities including:
+* Stakeholder mapping
+* Requirements gathering & alignment
+* User-centred design
+* Research output development roadmapping
+* Workshop delivery
+
+Learn more about our work and access our tools and insights [here](https://the-turing-way.netlify.app/collaboration/stakeholder-engagement.html).
+
+## What makes for an impactful RAM engagement?
+
+We consider multiple dimensions of a research project and team to prioritise engagement across Turing's research projects:
+* **Applicability**: How much potential does a research project have for real-world use?
+* **Capability**: To what extent does the team have the skills to translate their research to real-world use?
+* **Values**: How much does the team's approach to research align with our focus areas (e.g. interoperable open source tools, interdisciplinary collaboration, participatory co-creation)
+* **Practices**: How ready are the team to adopt RAM ways of working?
+ 
+![RAM rhombus-Simple diagram drawio](https://github.com/alan-turing-institute/research-application-management/assets/5104098/19d17aff-5caa-441f-8703-1497553b1ab3)
+
+## Examples of our work
+* Open sourcing the [Data Safe Haven](https://www.turing.ac.uk/research/research-projects/data-safe-havens-cloud) and creating [national standards and communuity for Trusted Research environments](https://github.com/sa-tre/satre-specification)
+* Integrating A/B street for [use in a Bristol City Council public consultation tool](https://www.turing.ac.uk/blog/street-smart-putting-neighbourhood-design-hands-bristol-residents) to empower residents to design and assess street-level interventions in their neighbourhoods
+* Delivering cross-sector workshops with government and industry stakeholders in 'Health' and 'Autonomy' fields on behalf of the [AI Standards Hub](https://aistandardshub.org/)
+* Creating [PitchFest at AI UK](https://www.turing.ac.uk/pitchfest-share-your-research-world) to upskill Early Career Researchers to communicate the impact of their research to a public audience
+* Co-authoring public-facing whitepapers on Turing's [Environment & Sustainability](https://www.turing.ac.uk/news/publications/tackling-climate-change-data-science-and-ai) and [Digital Twins](https://www.turing.ac.uk/news/publications/towards-ecosystems-connected-digital-twins-address-global-challenges) research
+
+## Get in Touch
+
+RAM engagements can begin at any time in the project lifecycle, and we encourage researchers to reach out to the RAM team as early as possible so that we can incorporate the user perspective and consider possible applications in the development of research outputs. To learn more about RAMs or how to apply RAM methodologies in your research project, get in touch with our team at RAM@turing.ac.uk.
```

Comment:
Do we want to highlight that earlier is better stronger? E.g. RAMs can come in whenever but we recommend embedding them in the research team from the very beginning if possible